README.md

```diff
@@ -382,10 +382,18 @@ const vttOutput = webvtt(result);
 
 # Text to Speech
 
+## Rest
+
 ```js
 const { result } = await deepgram.speak.request({ text }, { model: "aura-asteria-en" });
 ```
 
+## Websocket
```

Comment:
Can you check out the other websocket example. It's a little more comprehensive than just the connection.